heron_base/launch/base.launch

```diff
@@ -26,26 +26,40 @@
         <node ns="wireless" pkg="wireless_watcher" name="wireless_watcher" type="watcher_node" respawn="true" />
     </group>
 
-    <group if="$(eval arg('namespace') == '')">
-     <node pkg="um6" type="um6_driver" name="um6_driver" output="screen">
-       <param name="tf_ned_to_enu" value="true"/>
-       <param name="/um6_driver/port" value="$(optenv HERON_IMU_UM6_PORT /dev/clearpath/imu)" />
-       <remap from="imu/data" to="imu/data_raw" />
-       <param name="lt_model" value="true" />
-       <param name="linear_acceleration_stdev" value="0.38" />
-       <param name="frame_id" value="imu_link" />
-     </node>
+    <group unless="$(optenv HERON_IMU_UM6 0)">
+      <!-- newer herons use the microstrain CM5-25 IMU -->
+      <include filename="$(find ros_mscl)/launch/microstrain.launch">
+        <arg name="name"          value="cv5" />
+        <arg name="port"          value="$(optenv HERON_IMU_PORT /dev/microstrain)" /> <!-- TODO - verify the correct default -->
+        <arg name="imu_frame_id"  value="imu_link" />
+        <!-- TODO - double-check topic names -->
```

Comment:
Verify please

Reply:
Same as the other note to myself; the PR is WIP, but I'll address any memos to myself before final review.